projects/can_api/utils.py

```diff
@@ -1,14 +1,25 @@
+import pdb
+
+
 def get_rx_messages(subs, messages):
     """
-    Gets the received Messages from the list of messages, based on the subs 
+    Gets the received Messages from the list of messages, based on the subs
     dictionary.
     """
+    # Check to see if subs has a message not specified in the DBC
+    dbc_message_names = [dbc_message.name for dbc_message in messages]
```

Comment:
This is the check we talked about last commit about checking for if the subs has a message that is missing from the dbc or not - not sure if it belongs here or if it should be more upstream or not though.